src/scripts/worldmap/MapHelper.ts

```diff
@@ -18,10 +19,17 @@ class MapHelper {
     };
 
     public static accessToRoute = function (route: number, region: GameConstants.Region) {
-        if (!player.hasBadge(GameConstants.routeBadgeRequirements[region][route])) {
+        if (region == GameConstants.Region.johto) { // Unlock all of johto for testing
+            return true
+        }
```

Comment:
Now that all the requirements are added this should no longer be necessary right?

Reply:
I don't think routeRequirements has been filled out yet.